R/selections.R

```diff
@@ -94,7 +94,8 @@
 NULL
 
 
-eval_select_recipes <- function(quos, data, info) {
+eval_select_recipes <- function(quos, data, info, allow_rename = FALSE) {
```

Comment:
Since we plan to export this, and because I would like to match the `tidyselect::eval_select()` interface as much as possible, I think it would be nice to follow tidyselect's lead and add `...` between `info` and `allow_rename = FALSE`.

The dots would have no purpose, except for separating the required arguments from the optional ones. This is nice because it forces you to specify optional arguments by name, which is more robust and provides more information when you are just reading, but not running, the code. We would ensure that the user cannot pass any information in the `...` by calling `ellipsis::check_dots_empty()` as the first line of `eval_select_recipes()`, like tidyselect does. This would require Importing ellipsis, which is fine.

This design guide is full of tips like this
https://design.tidyverse.org/

This particular pattern is data/dots/details in
https://design.tidyverse.org/dots-position.html

See that `eval_select()` uses this pattern (except for the env argument, which is a bit special):

```r
> tidyselect:::eval_select
function (expr, data, env = caller_env(), ..., include = NULL, 
    exclude = NULL, strict = TRUE, name_spec = NULL, allow_rename = TRUE) 
{
    ellipsis::check_dots_empty()
    eval_select_impl(data, names(data), as_quosure(expr, env), 
        include = include, exclude = exclude, strict = strict, 
        name_spec = name_spec, allow_rename = allow_rename)
}
```

Reply:
done!